main.py

```
@@ -1,194 +1,233 @@
+        self.is_running = False
+        
+        # 미체결 주문 현황 출력
+        order_summary = self.order_manager.get_order_summary()
+        if order_summary['total_pending'] > 0:
+            self.logger.info(f"중지 시점 미체결 주문: {order_summary['total_pending']}개")
+            for order in order_summary['orders']:
+                self.logger.info(f"  - {order['symbol']} {order['type']} {order['quantity']}주 @ ${order['price']} ({order['time']})")
+        
+        # 오늘 거래 기록 요약 출력
+        transaction_summary = self.transaction_logger.get_summary()
+        self.logger.info(f"=== 오늘 거래 기록 요약 ===")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)